### ISSUE TITLE
[Bug]: .sst file in /volumes/milvus/rdb_data at six hours ago  , can't be auto deleted when I was set rocksmq.retentionTimeInMinutes to 180 。

### ISSUE THREAD

**Issue by 710585357** [milvus-io/milvus#23906]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: v2.2.3
- Deployment mode(standalone or cluster): standalone 
- MQ type(rocksmq, pulsar or kafka):    rocksmq
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): CentOS 7.9
- CPU/Memory: 4/16G
- GPU: null
- Others:
```

### Current Behavior

.sst file in /volumes/milvus/rdb_data at six hours ago  , can't be auto deleted when I was set rocksmq.retentionTimeInMinutes to 180 ,and restart the standalone .

### Expected Behavior

The .sst file will be automatically deleted when it expires

### Steps To Reproduce

```markdown
1. cp milvus.yaml from standalone (docker) to local dir.
2. vi milvus.yaml and set rocksmq.retentionTimeInMinutes 180
3. cp local milvus.yaml to standalone (docker)
4. restart standalone (docker)
```

### Milvus Log

.sst file: 
https://github.com/710585357/demofile/raw/main/010377.zip

### Anything else?

no

**Comment by xiaofan-luan** [comment]
the compaction takes at least one day to take affect, because it will wait for compaction happned.
Tune rocksmq.compactionInterval to smaller value may trigger compaction to happen earlier.

I  would suggest at least have double disk space for the amount of data you want to write into milvus, and 50GB diskspace.

**Comment by xiaofan-luan** [comment]
should be fixed on 2.2.9 or later verison
